pytorch_lightning/callbacks/quantization.py

```diff
@@ -161,9 +184,21 @@ def __init__(
         self.modules_to_fuse = modules_to_fuse
         self._input_compatible = input_compatible
         self._convert_on_fit_end = quantize_on_fit_end
+
+        disable_observers = set(disable_observers)
+        unsupported_stages = disable_observers - set(self.ALL_DISABLE_OBSERVER_STAGES)
+        if unsupported_stages:
+            raise MisconfigurationException(
+                f'Unsupported stages "{tuple(sorted(unsupported_stages))}", allowed are '
+                f"{self.ALL_DISABLE_OBSERVER_STAGES}."
```

Comment:
this will print the tuple. if this is undesired, could do `", ".join(self.ALL_DISABLE_OBSERVER_STAGES)`

Reply:
The error message for `observer_type` used `tuple`.
https://github.com/PyTorchLightning/pytorch-lightning/blob/eaa16c7480ef6728d69113d6305014d39c2efece/pytorch_lightning/callbacks/quantization.py#L151

Should we follow it or change errors messages for both `observer_type` and `disable_observers` in this PR?

Reply:
no preference, just wanted to make sure we are aware what it prints.